cmd/oras/internal/option/target.go

```diff
@@ -231,10 +232,10 @@ func (opts *Target) NewReadonlyTarget(ctx context.Context, common Common, logger
 	return nil, fmt.Errorf("unknown target type: %q", opts.Type)
 }
 
-// EnsureReferenceNotEmpty ensures whether the tag or digest is empty.
-func (opts *Target) EnsureReferenceNotEmpty() error {
+// EnsureReferenceNotEmpty ensures that the reference of a manifest is not empty.
```

Comment:
Suggest to remove the manifest since the check might be done on a blob.
```suggestion
// EnsureReferenceNotEmpty returns formalized error when the reference is empty.
```

Reply:
Forgot to change the doc comment when refactoring the function.